src/Test.UnitTests.Sarif.Multitool/MergeCommandTests.cs

```diff
@@ -24,13 +24,16 @@ namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
     // requires one or more input files, we'll have to address this. We will probably have to
     // factor the output file writing and comparison infrastructure out of FileDiffingUnitTests.
     // We might be able to push it into a lower base class.
-    public class MergeCommandTests : FileDiffingUnitTests
+    public class MergeCommandTests : FileDiffingUnitTests, IClassFixture<MergeCommandTests.MergeCommandTestsFixture>
     {
+        public class MergeCommandTestsFixture : DeletesOutputsDirectoryOnClassInitializationFixture { }
+
         protected override string TestLogResourceNameRoot => "Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool.TestData." + TypeUnderTest;
 
         public MergeCommandTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
 
         [Fact]
+        [Trait(TestTraits.Bug, "1592")]
```

Comment:
Why are we adding this? If you want a reference to a bug, I suggest just adding a code comment with a clickable link. Using the attribute doesn't provide this value.

Reply:
The benefit of `[Trait]` is that you can use it to select tests to run from the command line. We can have the best of both worlds (clickable link + command line access) like this:

```
[Trait(TestTraits.Bug, "[https://github.com/microsoft/sarif-sdk/1340](https://github.com/microsoft/sarif-sdk/1340)")]
```

... which is what I did for all 15 occurrences in the code base.

---
In reply to: [316738027](https://github.com/microsoft/sarif-sdk/pull/1651#discussion_r316738027) [](ancestors = 316738027)

Reply:
I'll send mail to propose this to the team.

---
In reply to: [316773161](https://github.com/microsoft/sarif-sdk/pull/1651#discussion_r316773161) [](ancestors = 316773161,316738027)